src/macros.rs

```diff
@@ -5,7 +5,7 @@
 #[macro_export]
 macro_rules! nlimbs {
     ($bits:expr) => {
-        (($bits + $crate::Limb::BITS - 1) / $crate::Limb::BITS) as usize
+        u32::div_ceil($bits, $crate::Limb::BITS) as usize
```

Comment:
Note: can't use the following due to type inference problems:

```rust
$bits.div_ceil($crate::Limb::BITS) as usize
```

It causes the following error:

```
error[E0689]: can't call method `div_ceil` on ambiguous numeric type `{integer}`
   --> src/macros.rs:8:15
    |
8   |           $bits.div_ceil($crate::Limb::BITS) as usize
    |                 ^^^^^^^^
```